File: DevExpress-Demo/CS/ChartsDemo.Wpf/DataModels/OutsideVendorCosts.cs
using System;
using System.Data;

namespace ChartsDemo {

    public class OutsideVendorCosts {
        public DataTable Data {
            get { return GetData(); }
        }

        public DataTable GetData() {
            int lastYear = DateTime.Now.Year - 1;
            DataTable table = new DataTable();
            table.Columns.AddRange(new DataColumn[] { new DataColumn("Year", typeof(DateTime)), new DataColumn("Company", typeof(string)), new DataColumn("Costs", typeof(decimal)) });

            table.Rows.Add(new DateTime(lastYear - 6, 12, 31), "DevAV North", 362.5M);
            table.Rows.Add(new DateTime(lastYear - 5, 12, 31), "DevAV North", 348.4M);
            table.Rows.Add(new DateTime(lastYear - 4, 12, 31), "DevAV North", 279.0M);
            table.Rows.Add(new DateTime(lastYear - 3, 12, 31), "DevAV North", 230.9M);
            table.Rows.Add(new DateTime(lastYear - 2, 12, 31), "DevAV North", 203.5M);
            table.Rows.Add(new DateTime(lastYear - 1, 12, 31), "DevAV North", 197.1M);

            table.Rows.Add(new DateTime(lastYear - 6, 12, 31), "DevAV South", 277.0M);
            table.Rows.Add(new DateTime(lastYear - 5, 12, 31), "DevAV South", 328.5M);
            table.Rows.Add(new DateTime(lastYear - 4, 12, 31), "DevAV South", 297.0M);
            table.Rows.Add(new DateTime(lastYear - 3, 12, 31), "DevAV South", 255.3M);
            table.Rows.Add(new DateTime(lastYear - 2, 12, 31), "DevAV South", 173.5M);
            table.Rows.Add(new DateTime(lastYear - 1, 12, 31), "DevAV South", 131.8M);

            return table;
        }
    }

}

File: DevExpress-Demo/VB/ChartsDemo.Wpf/DataModels/OutsideVendorCosts.vb
Imports System
Imports System.Data

Namespace ChartsDemo

    Public Class OutsideVendorCosts

        Public ReadOnly Property Data As DataTable
            Get
                Return GetData()
            End Get
        End Property

        Public Function GetData() As DataTable
            Dim lastYear As Integer = Date.Now.Year - 1
            Dim table As DataTable = New DataTable()
            table.Columns.AddRange(New DataColumn() {New DataColumn("Year", GetType(Date)), New DataColumn("Company", GetType(String)), New DataColumn("Costs", GetType(Decimal))})
            table.Rows.Add(New DateTime(lastYear - 6, 12, 31), "DevAV North", 362.5D)
            table.Rows.Add(New DateTime(lastYear - 5, 12, 31), "DevAV North", 348.4D)
            table.Rows.Add(New DateTime(lastYear - 4, 12, 31), "DevAV North", 279.0D)
            table.Rows.Add(New DateTime(lastYear - 3, 12, 31), "DevAV North", 230.9D)
            table.Rows.Add(New DateTime(lastYear - 2, 12, 31), "DevAV North", 203.5D)
            table.Rows.Add(New DateTime(lastYear - 1, 12, 31), "DevAV North", 197.1D)
            table.Rows.Add(New DateTime(lastYear - 6, 12, 31), "DevAV South", 277.0D)
            table.Rows.Add(New DateTime(lastYear - 5, 12, 31), "DevAV South", 328.5D)
            table.Rows.Add(New DateTime(lastYear - 4, 12, 31), "DevAV South", 297.0D)
            table.Rows.Add(New DateTime(lastYear - 3, 12, 31), "DevAV South", 255.3D)
            table.Rows.Add(New DateTime(lastYear - 2, 12, 31), "DevAV South", 173.5D)
            table.Rows.Add(New DateTime(lastYear - 1, 12, 31), "DevAV South", 131.8D)
            Return table
        End Function
    End Class
End Namespace
